laws/src/main/scala/org/http4s/laws/discipline/ArbitraryInstances.scala

```diff
@@ -634,11 +638,12 @@ private[http4s] trait ArbitraryInstances {
     } yield Uri.Authority(maybeUserInfo, host, maybePort)
   }
 
-  val genPctEncoded: Gen[String] =
+  lazy val genPctEncoded: Gen[String] =
```

Comment:
A long, long time ago, I saw lazy arbitraries making a difference in a profile, which caused us to use eager vals.  I don't know whether that's still a concern, but I think you could just by moving them above the new use.

Whether eager values are smart in a trait is a separate question.

I'm fine either way.